core/templates/Polyfills.ts

```diff
@@ -159,3 +159,17 @@ if (navigator.mediaDevices.getUserMedia === undefined) {
     });
   };
 }
+
+// Create a polyfill for Object.entries() for Chrome.
+if (!Object.entries) {
+  Object.entries = function(obj) {
```

Comment:
Can we use array function and also specify the type of function parameter.

Reply:
@ashutoshc8101 `Object` is a group of key value pairs, so the type for the function parameter would be `[x: string]: any`, right? But es-lint rules don't allow `any` types.

Reply:
It should be obj: object.

Reply:
Done